rskj-core/src/main/java/co/rsk/peg/PendingFederation.java

```diff
@@ -21,9 +21,9 @@
 import co.rsk.bitcoinj.core.BtcECKey;
 import co.rsk.config.BridgeConstants;
 import co.rsk.crypto.Keccak256;
-import co.rsk.peg.bitcoin.ErpRedeemScriptBuilder;
-import co.rsk.peg.bitcoin.NonStandardErpRedeemScriptBuilderFactory;
-import co.rsk.peg.bitcoin.P2shErpRedeemScriptBuilder;
+import co.rsk.peg.federation.Federation;
```

Comment:
Should this class also be moved to federation package? What do you think?

Reply:
i thought that too! but it uses `BridgeSerializationUtils` class in `getHash` method